app/src/views/Search.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <div class="flex h-full w-full min-h-0">
+    <AppBar v-model:search-input="searchInput" />
+
+    <div class="flex flex-col mt-16 mx-2 w-full overflow-y-auto">
+      <div v-if="searchResults.length === 0" class="m-auto max-w-52 text-center text-xl">
+        <p>Suche nach einer Haltestelle oder einem Fahrzeug</p>
+      </div>
+      <router-link
+        v-for="searchResult in searchResults"
+        :key="searchResult.refIndex"
+        :to="{ name: 'map-marker', params: { markerType: searchResult.item.type, markerId: searchResult.item.id } }"
```

Comment:
@lukashass What's your suggestion to map `searchResult.item.type` to `stop` or `vehicle` ...?